clients/admin-ui/src/features/common/ConfirmationModal.tsx

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ThemingProps } from "@chakra-ui/system";
```

Comment:
We don't want to add any chakra ui dependencies directly to the admin ui to avoid weird dependency issues. This component will be moved to to `fidesui` soon so suppressing this lint is temporary.